src/tls_link.c

```diff
@@ -143,6 +148,13 @@ static void tls_read_cb(uv_link_t *l, ssize_t nread, const uv_buf_t *b) {
             inlen = 0;
         } while (rc == TLS_MORE_AVAILABLE && out_bytes > 0);
 
+        if (rc == TLS_HAS_WRITE) {
+            uv_buf_t buf;
+            buf.base = malloc(32 * 1024);
+            int tls_rc = tls->engine->api->write(tls->engine->engine, NULL, 0, buf.base, &buf.len, 32 * 1024);
```

Comment:
nit. probably should be a constant. someone might miss that 32 * 1024 is there twice